fedimint-client/src/lib.rs

```diff
@@ -341,7 +341,7 @@ pub trait IGlobalClientContext: Debug + MaybeSend + MaybeSync + 'static {
         kind: EventKind,
         module: Option<(ModuleKind, ModuleInstanceId)>,
         payload: serde_json::Value,
-        transient: bool,
+        persist: bool,
```

Comment:
This change could have been separated out. I'm also quite confused how the name can suddenly be the opposite but no other logic is inverted?

Reply:
It is a bug that I noticed now. :D